orcoursetrion/actions/github.py

```diff
@@ -38,3 +38,45 @@ def create_export_repo(course, term, description=None):
         config.ORC_STUDIO_ORG, repo_name, config.ORC_STUDIO_DEPLOY_TEAM
     )
     return repo
+
+
+def create_xml_repo(course, term, team, description=None):
+    """Creates a course repo at
+    :py:const:`~orcoursetrion.config.ORC_GH_API_URL` with key
+    :py:const:`~orcoursetrion.config.ORC_GH_OAUTH2_TOKEN` and at
+    organization :py:const:`~orcoursetrion.config.ORC_XML_ORG`, and
+    with ``team`` as a collaborator (Along with
+    :py:const:`~orcoursetrion.config.ORC_XML_DEPLOY_TEAM`).
+
+    This also adds a github Web hook to the course development
+    environment `gitreload <https://github.com/mitodl/gitreload>`_
+    server via
+    :py:const:`~orcoursetrion.config.ORC_STAGING_GITRELOAD`.
+
+    Args:
+        course (str): Course name to be used to name repo (i.e. 6.004r)
+        term (str): Term the course is expected to run (i.e. 2015_Spring)
+        team (str): Name of the team to add read/write access to this repo.
```

Comment:
Consider changing "Name of the team" to "Name of the existing team" to indicate that the team must already exist.